### PR TITLE
[mod_say_fr] Fix "10" not playing in French.

### PR DESCRIPTION
**Pull request by dvillaume**

### CODE REVIEW

**Comment by andywolk** [comment]
Please use a proper commit message. Here is the example: `[mod_say_fr] Fix "10" not playing in French.`

**Comment by dvillaume** [comment]
@andywolk you're true, that's my first commit, i'll do better the next time.

**Comment by andywolk** [comment]
You changed title of the Pull Request. Please change commit message: https://help.github.com/en/articles/changing-a-commit-message

File: src/mod/say/mod_say_fr/mod_say_fr.c [hunk 1]
```diff
@@ -119,7 +119,7 @@ static switch_status_t play_group(switch_say_method_t method, switch_say_gender_
 		}
 	}
 
-	if (c || ((ftdNumber == 1 && (a || b || c)) && (a && (b || c)))) {
+	if (c || ((ftdNumber == 1 && (a || b || c)) && (a && (b || c))) || (a==0 && b==1 && c==0) ) {
```
Comment (dragos-oancea) [inline src/mod/say/mod_say_fr/mod_say_fr.c]: use FS indentation (use spaces), eg: (a == 0 && b == 1 && c == 0)

Comment (dvillaume) [reply src/mod/say/mod_say_fr/mod_say_fr.c]: Is that better to you ?

**Comment by andywolk** [comment]
Please squash two commits into one.

**Comment by dvillaume** [comment]
Done.

**Pull request merged**